src/main/webapp/intro.html

```diff
@@ -170,23 +175,6 @@ <h4 class="card-title">Cis Xing</h4>
                   <div class="card-body text-center mt-4">
                     <h4 class="card-title">Cis Xing</h4>
                     <p class="card-text">This is a introduction part for cis for write blablablablabla.....blablablablablabla. IAmAPlaceholder IAmAPlaceholder. IAmAPlaceholder IAmAPlaceholder IAmAPlaceholder. IAmAPlaceholder IAmAPlaceholder IAmAPlaceholder.</p>
```

Comment:
You can use my profile photo.
One sentence about me: Software engineer who struggles with deadlocks.
Description: Cis is a software engineer who only recently watched the classic 1999 movie "the Matrix" and now she is obsessed with Keanu Reeves.